src/utils/passportStrategyCallback.js

```diff
@@ -0,0 +1,31 @@
+import models from '../models';
+import config from '../config';
+/**
+   * It uses the done method to return an authenticated user or an error object
+   * @function strategyCallback
+   * @param {string} accessToken
+   * @param {string} refreshToken
+   * @param {Object} profile
+   * @param {function} done
+   * @returns {Object} (user | error)
+*/
+const strategyCallback = async (accessToken, refreshToken, profile, done) => {
+  try {
+    if (!profile) return done(null, undefined);
+    const email = profile.emails[0].value;
+    const existingUser = await models.Users.findOne({ where: { email } });
+    if (existingUser) {
+      return done(null, existingUser);
+    }
+    const { name: { familyName: lastName, givenName: firstName } } = profile;
+    const password = config.password;
```

Comment:
You need to hash the password before saving it to the database

Reply:
Done.